build/__rubyVersions.sh

```diff
@@ -4,4 +4,4 @@ GEM_VERSION='3.1.4'
 RUBY27_VERSION='2.7.1'
 RUBY27_TAR_SHA256='b224f9844646cc92765df8288a46838511c1cec5b550d8874bd4686a904fcee7'
 RUBY26_VERSION='2.6.6'
-RUBY26_TAR_SHA256='5db187882b7ac34016cd48d7032e197f07e4968f406b0690e20193b9b424841f'
\ No newline at end of file
+RUBY26_TAR_SHA256='5db187882b7ac34016cd48d7032e197f07e4968f406b0690e20193b9b424841f'
```

Comment:
why is this change?

Reply:
Probably happened during rebase but if you notice the change is only addition of a new line